src/Core/src/Platform/Windows/TextBoxExtensions.cs

```diff
@@ -119,5 +119,17 @@ public static void UpdateMaxLength(this MauiTextBox textBox, IEntry entry)
 			if (currentControlText.Length > maxLength)
 				textBox.Text = currentControlText.Substring(0, maxLength);
 		}
+
+		public static void UpdateAutoSize(this MauiTextBox textBox, IEditor editor)
+		{
+			if (editor.AutoSize == EditorAutoSizeOption.Disabled)
+			{
+				textBox.AcceptsReturn = false;
```

Comment:
I don't think this is the correct approach for this. The Editor should always be set to AcceptsReturns = true

For example if the user forces the height of the editor to "500" then the editor should still accept returns 

I think what you might be running up against here are some layout behavior that's not implemented yet

Which is why things aren't resizing for you